src/RNCamera.js

```diff
@@ -551,6 +551,10 @@ export default class Camera extends React.Component<PropsType, StateType> {
     }
   }
 
+  async getSupportedPreviewFpsRange() {
```

Comment:
```suggestion
  getSupportedPreviewFpsRange = async (): number => {
```

Reply:
@MateusAndrade thanks a lot. I implemented like this 

```javascript
  getSupportedPreviewFpsRange = async (): Promise<[]> => {
    return await CameraManager.getSupportedPreviewFpsRange(this._cameraHandle);
  };
```
and changed the android api to return `integer` instead of `string` (using `putInt` instead of `putString`).

```java
  @ReactMethod
  public void getSupportedPreviewFpsRange(final int viewTag, final Promise promise) {
      final ReactApplicationContext context = getReactApplicationContext();
      UIManagerModule uiManager = context.getNativeModule(UIManagerModule.class);
      uiManager.addUIBlock(new UIBlock() {
          @Override
          public void execute(NativeViewHierarchyManager nativeViewHierarchyManager) {
              final RNCameraView cameraView;

              try {
                  cameraView = (RNCameraView) nativeViewHierarchyManager.resolveView(viewTag);
                  WritableArray result = Arguments.createArray();
                  ArrayList<int[]> ranges = cameraView.getSupportedPreviewFpsRange();
                  for (int[] range : ranges) {
                      WritableMap m = new WritableNativeMap();
                      m.putInt("MAXIMUM_FPS", range[0]);
                      m.putInt("MINIMUM_FPS", range[1]);
                      result.pushMap(m);
                  }
                  promise.resolve(result);
              } catch (Exception e) {
                  e.printStackTrace();
              }
          }
      });
  }
```
should I implement a more stringent check on the `javascript` returned value (insted of `Promise<[]>`)? This is an example of the value returned once promise is fullfilled:

```json
[ 
  { MINIMUM_FPS: 15000, MAXIMUM_FPS: 15000 },
  { MINIMUM_FPS: 20000, MAXIMUM_FPS: 20000 },
  { MINIMUM_FPS: 24000, MAXIMUM_FPS: 24000 },
  { MINIMUM_FPS: 30000, MAXIMUM_FPS: 15000 },
  { MINIMUM_FPS: 30000, MAXIMUM_FPS: 30000 } 
]
```

I could try to find more a more appropriate [advanced type](https://www.typescriptlang.org/docs/handbook/advanced-types.html) and include it in the `types/index.d.ts`. 

Thanks a lot

Reply:
I think we are good to go with a simple `integer` return from this call. Or using something like a type or a tuple(i used both on TS 😅 ). But, for me this looks fine.